doc/classes/GraphEdit.xml

```diff
@@ -48,8 +48,8 @@
 				Below is a sample code to help get started:
 				[codeblock]
 				func _is_in_output_hotzone(in_node, in_port, mouse_position):
-				    var port_size: Vector2 = Vector2(get_theme_constant("port_grab_distance_horizontal"), get_theme_constant("port_grab_distance_vertical"))
-				    var port_pos: Vector2 = in_node.get_position() + in_node.get_connection_output_position(in_port) - port_size / 2
+				    var port_size : Vector2 = Vector2(get_theme_constant("port_grab_distance_horizontal"), get_theme_constant("port_grab_distance_vertical"))
+				    var port_pos : Vector2 = in_node.get_position() + in_node.get_port_output_position(in_port) - port_size / 2
```

Comment:
https://docs.godotengine.org/en/latest/tutorials/scripting/gdscript/gdscript_styleguide.html#static-typing

Reply:
Thanks for the review. This isn't really a part of this PR. It's from a commit this PR is based on. The issue was fixed in the original commit, this PR just needs to be rebased now.